Rakefile

```diff
@@ -1,7 +1,20 @@
+require 'rubygems'
+require 'bundler/setup'
 require 'bundler/gem_tasks'
 require 'rake/testtask'
 require 'rubocop/rake_task'
 
+# Immediately sync all stdout
```

Comment:
Ah sorry, that was me comparing the Rakefile with the one I cooked up for the landrush-ip repository because it wouldn't work with my OS X' Rake. These changes aren't necessary at all.